R/pkg/R/client.R

```diff
@@ -42,6 +42,19 @@ determineSparkSubmitBin <- function() {
   }
   sparkSubmitBinName
 }
+# R supports both file separator in the file path (Unix : / and Windows: \) irrespective of the operating system
+# but when passing file path to Java or script program, it has to be converted according to operating system
+
+determinefileSeperator <- function() {
+  if (.Platform$OS.type == "unix") {
```

Comment:
According to https://stat.ethz.ch/R-manual/R-devel/library/base/html/Platform.html, .Platform$OS.type have only two options: "unix" or "windows".